.circleci/config.yml

```diff
@@ -95,7 +95,7 @@ commands:
 jobs:
   build:
     docker:
-      - image: argoproj/argocd-test-tools:v0.2.0
+      - image: argoproj/argocd-test-tools:v0.3.0
```

Comment:
`argocd-test-tools:v0.3.0` has `kustomize2` removed, because it also upgrades toolchain Kustomize to v3 only (see  #3347)